### DISCUSSION TITLE
Progress bar on neutral background is not visible in cupcake theme

### DISCUSSION THREAD

**Discussion by t1u1** [saadeghi/daisyui#3310] [Q&A]
Repro: https://play.tailwindcss.com/wWndQ9FxeZ

I am not sure if this is a bug in the cupcake theme, or some additional color classes need to be specified. Please help!

**Comment by saadeghi** [comment]
Fixed in `4.12.21`  
Thanks for reporting the problem!

**Comment by t1u1** [comment]
Thanks for the quick fix! :+1: 

(I seemed to have shared the default play link for the repro, but glad you figured it out nonetheless )

**Comment by t1u1** [comment]
I managed to recover my test case: 
https://play.tailwindcss.com/CQl6qx0LNk

While the "completion" part is now visible, the non-complete part of the progress bar is still not visible in "cupcake" and "light" themes.

It would be nice if 100/200/300 variants of the colors were available for each named color. For example, bg-neutral-100, or primary-200, etc. Then progress bar could use "bg-neutral-100".
